source/channels.yaml

```diff
@@ -410,6 +410,44 @@
               additionalProperties:
                 $ref: "#/definitions/ChannelMember"
 
+  '/teams/{team_id}/channels/{channel_id}/update_member_roles':
```

Comment:
To get a channel member, the current API is `/teams/{team_id}/channels/{channel_id}/members/{user_id}` so maybe go with `/teams/{team_id}/channels/{channel_id}/members/{user_id}/update_roles`? Also matches `/users/{user_id}/update_roles` as well